File: inventory-service/src/main/java/com/orderproduct/inventoryservice/common/exception/InternalServerException.java
package com.orderproduct.inventoryservice.common.exception;

import org.springframework.http.HttpStatus;

public class InternalServerException extends ApiException {

    public InternalServerException() {
        super(
                HttpStatus.INTERNAL_SERVER_ERROR,
                ErrorComponent.SOMETHING_WENT_WRONG_ERROR_CODE,
                ErrorComponent.somethingWentWrongMsg);
    }
}

File: product-service/src/main/kotlin/com/orderproduct/productservice/common/InternalServerException.kt
package com.orderproduct.productservice.common

import org.springframework.http.HttpStatus

class InternalServerException : ApiException(
    httpStatus = HttpStatus.INTERNAL_SERVER_ERROR,
    errorCode = ErrorComponent.SOMETHING_WENT_WRONG_ERROR_CODE,
    errorMessage = ErrorComponent.somethingWentWrongMsg
)
